vmdb/app/helpers/application_helper.rb

```diff
@@ -2706,6 +2708,7 @@ def render_listnav_filename
     elsif %w(offline retired templates vm vm_cloud vm_or_template).include?(@layout)
       "vm"
     elsif %w(action availability_zone cim_base_storage_extent cloud_tenant condition container_group
+             container_route container_project
```

Comment:
@abonas, yes, if you have any filters to show for a CI on list view screen then it should be included in the list of CIs in the if and elsif part is to load lisnav on the the summary screen